app/code/Magento/CatalogInventory/Api/RegisterProductSaleInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogInventory\Api;
+
+use Magento\CatalogInventory\Api\Data\StockItemInterface;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * @api
+ */
+interface RegisterProductSaleInterface
+{
+    /**
+     * Subtract product qtys from stock
+     * Return array of items that require full save
+     *
+     * Method signature is unchanged for backward compatibility
```

Comment:
Some reference to the item which backward compatibility is preserved for may be added here.